TACC/Core-CMS-Custom
# task/WP-557: Cleaning up unused database fields + create submitter code

## Overview
UTHealth notified us about removing now-unused fields on the `submitters` and `registrations` tables - this PR removes any references to said fields.
…

## Related

- [WP-557](https://tacc-main.atlassian.net/browse/WP-557)
…

## Changes
- Removed read and write of `applicable_period_start` and `applicable_period_end` from `get_registrations` and `create_registration` database functions
- Re-indexed registration data formatting to compensate for removal of above fields
- Removed `create_submitter` database function + connected view function (all have been disabled in portal for over a year)
…

## Testing

_* No change in functionality, just testing for regressions *_
1. Go to https://localhost:8000/register/request-to-submit
2. Fill out a registration and submit and confirm no error on submission
3. Go to https://localhost:8000/administration/list-registration-requests and confirm all info given on this record appears on the table and in the view + edit record actions

## UI

…
